src/views/TeamMember.vue

```diff
@@ -18,156 +19,46 @@ const getTeamMemberName = () => {
+
 onMounted(() => {
   getTeamMemberName()
 })
 </script>
 
 <template>
-  <!-- TODO: Add loader -->
   <span v-if="!member">Loading...</span>
   <div v-else class="team">
-    <div class="container">
-      <div class="content">
-        <h1 data-test="member-page-name" class="head-1">
-          {{ $t(`team.${member}.name`) }}
-        </h1>
-        <div>
-          <div
-            class="photo"
-            :style="`background-image: url(${$t(`team.${member}.image`)});`"
-            data-test="member-page-photo"
-          />
-        </div>
-        <div class="socialIcons !p-2">
-          <a
-            v-if="$t(`team.${member}.github_url`).length > 1"
-            :href="$t(`team.${member}.github_url`)" target="_blank"
-            data-test="member-page-github"
-          >
-            <i class="mobile-menu-icon fab fa-github" />
-          </a>
-          <a
-            v-if="$t(`team.${member}.linkedin_url`).length > 1"
-            :href="$t(`team.${member}.linkedin_url`)" target="_blank"
-            data-test="member-page-linkedin"
-          >
-            <i class="mobile-menu-icon fab fa-linkedin" />
-          </a>
-          <a
-            v-if="$t(`team.${member}.twitter_url`).length > 1"
-            :href="$t(`team.${member}.twitter_url`)" target="_blank"
-            data-test="member-page-twitter"
-          >
-            <i class="mobile-menu-icon fab fa-twitter" />
-          </a>
-          <a
-            v-if="$t(`team.${member}.website`).length > 1"
-            :href="$t(`team.${member}.website`)" target="_blank"
-            data-test="member-page-website"
-          >
-            <i class="mobile-menu-icon fa fa-cloud" />
-          </a>
-        </div>
-
-        <!-- TODO: Make this a proper <p> -->
+    <div class="container p-32px max-w-700px m-auto text-center flex flex-col justify-center items-center gap-16px">
+      <h1 data-test="member-page-name" class="text-2rem font-700">
+        {{ $t(`team.${member}.name`) }}
+      </h1>
+      <div>
         <div
-          class="description !p-2"
-          data-test="member-page-description"
-          v-html="$t(`team.${member}.description`)"
+          :data-test="`team-member-${member}-index-photo`"
+          class="block w-128px h-128px rounded-full bg-center bg-cover"
+          :style="`background-image: url( ${$t(`team.${member}.image`)} );`"
         />
-
-        <router-link
-          class="user-profile-link"
-          :to="{ name: 'Team' }" data-test="nav-team-page-link"
-        >
-          {{ t('redirect.back') }}
-        </router-link>
       </div>
+
+      <SocialIcons :member="member" :socials="socials" />
+
+      <p
+        class=""
+        data-test="member-page-description"
+        v-html="$t(`team.${member}.description`)"
+      />
+
+      <router-link
+        class="w-fit px-15px py-5px border-1px border-solid border-transparent rounded-full bg-#586379 text-#fff text-1rem font-700 hover:bg-#2e3440"
```

Comment:
Unify classes without utilizing `px` and hex colours in line. See feedback left on other comments

Reply:
[fix(components): align unocss classes to project, manage light-dark m…](https://github.com/Schrodinger-Hat/schrodinger-hat-website/pull/94/commits/bd948059b85338410d2e8fe8a3c20a45a6cd6112) 
fixed